test/unittesting/testingutils.go

```diff
@@ -0,0 +1,624 @@
+			if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " Expires" {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " expires" {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else {
+				idRefreshTokenExpiry = strings.Split(strings.Split(cookie, ";")[3], "=")[1]
+			}
+			if strings.Split(strings.Split(cookie, ";")[1], "=")[0] == " Path" {
+			}
+			for _, property := range strings.Split(cookie, ";") {
+				if strings.Index(property, "HttpOnly") == 1 {
+					idRefreshTokenHttpOnly = "true"
+					break
+				}
+			}
+		} else if strings.Split(strings.Split(cookie, ";")[0], "=")[0] == "sAccessToken" {
+			accessToken = strings.Split(strings.Split(cookie, ";")[0], "=")[1]
+			if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " Expires" {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else if strings.Split(strings.Split(cookie, ";")[2], "=")[0] == " expires" {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[2], "=")[1]
+			} else {
+				accessTokenExpiry = strings.Split(strings.Split(cookie, ";")[3], "=")[1]
+			}
+			if strings.Split(strings.Split(cookie, ";")[1], "=")[0] == " Path" {
+			}
+			for _, property := range strings.Split(cookie, ";") {
+				if strings.Index(property, "HttpOnly") == 1 {
+					accessTokenHttpOnly = "true"
+					break
+				}
+			}
+		}
+	}
+	return map[string]string{
+		"sAccessToken":           accessToken,
+		"sRefreshToken":          refreshToken,
+		"sIdRefreshToken":        idRefreshTokenFromCookie,
+		"refreshTokenExpiry":     refreshTokenExpiry,
+		"refreshTokenDomain":     refreshTokenDomain,
+		"refreshTokenHttpOnly":   refreshTokenHttpOnly,
+		"idRefreshTokenExpiry":   idRefreshTokenExpiry,
+		"idRefreshTokenDomain":   idRefreshTokenDomain,
+		"idRefreshTokenHttpOnly": idRefreshTokenHttpOnly,
+		"accessTokenExpiry":      accessTokenExpiry,
+		"accessTokenDomain":      accessTokenDomain,
+		"accessTokenHttpOnly":    accessTokenHttpOnly,
+	}
+}
+
+func getInstallationDir() string {
+	installationDir := os.Getenv("INSTALL_DIR")
+	if installationDir == "" {
+		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(false) + "supertokens-root"
+	} else {
+		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(true) + installationDir
+	}
+	return installationDir
+}
+
+func SetKeyValueInConfig(key string, value string) {
+	installationPath := getInstallationDir()
+	pathToConfigYamlFile := installationPath + "/config.yaml"
+	f, err := os.OpenFile(pathToConfigYamlFile, os.O_APPEND|os.O_WRONLY, 0600)
+	if err != nil {
+		panic(err)
+	}
+
+	defer f.Close()
+
+	f.Write([]byte("\n" + key + ": " + value))
+
+	if _, err = f.WriteString(key + ": " + value + "\n"); err != nil {
+		panic(err)
+	}
+}
+
+func SetKeyAndNumberValueInConfig(key string, value int) {
```

Comment:
I don't think you need this function as in the config file, adding a number is just like adding a string. So you can replace calls to this with calls to `SetKeyValueInConfig` and convert the number to a string when calling the function?